### DISCUSSION TITLE
Getting Unable to parse entity error when trying to send message

### DISCUSSION THREAD

**Discussion by celproject** [bbernhard/signal-cli-rest-api#395] [Q&A]
Hello, I am trying to send message via signal API. Initially I am able to send a message first time, but after that it throws an error : 

POST https://signal.autosrealm.com/v2/send
Payload : {
  "message": "Vehicle 2018 MERCEDES-BENZ GLC-Class - WDC0J6EB0JF411174 was updated.\n              Check out more at:  https://autosrealm.com/#/search-vehicle/WDC0J6EB0JF411174",
  "number": "+359887676214",
  "recipients": ["+359878101410"]
}
Error: 
{
    "error": "INFO  AccountHelper - The Signal protocol expects that incoming messages are regularly received.\nWARN  SendHelper - Failed to send message due to IO exception: Unable to parse entity\nFailed to send message\n"
}

**Comment by bbernhard** [comment]
Do you notice any patterns? Does it happen with a particular JSON payload? If you retry a failed attempt, does it work then? Which mode (normal, native, json-rpc) do you use?

**Comment by celproject** [comment]
Hi it did it in native, we are testing now in json-rpc and looks ok for now


Automotive Import / Export
USA AND CANADA

Tsvetan Mitev
+359887676214


________________________________
